api/src/controllers/Invoices.js

```diff
@@ -12,8 +14,12 @@ export default class InvoiceController {
   }
 
   async getByTxId(req, res) {
-    // req.params.txid
-    throw new Error('Not implemented');
+    const inv = await models.invoice.findByPk(req.params.txid);
+    if (inv) {
+      res.json(serializers.invoice(inv));
+    } else {
+      res.status(404).send('Not found');
```

Comment:
Formato

Reply:
Consertado para o formato:
`{
    "error": {
        "type": 404,
        "message": "Not Found"
    }
}`